src/client/components/pages/RoomAdmin/RoomAdmin.tsx

```diff
@@ -308,11 +304,11 @@ const RoomAdmin: FunctionComponent = (): ReactElement => {
                     </TableHeadingCell>
                     <TableHeadingCell scope="col">
                       <TextInput
-                        id="filterByCapacityName"
-                        name="filterByCapacityName"
+                        id="capacity-filter"
+                        name="capacity-filter"
                         placeholder="Enter Capacity"
                         value={capacityFilter}
-                        label="The table will be filtered by entered capacity"
+                        label="Change to filter the list of meetings by capacity"
```

Comment:
Same for this label - the room admin is not related to meetings, so this label text should be updated.